ui/job-view/pushes/Revision.jsx

```diff
@@ -47,16 +50,43 @@ export class Revision extends React.PureComponent {
     super(props);
     const { revision } = this.props;
 
+    this.state = {
+      tooltipOpen: {},
+    };
+
     // eslint-disable-next-line prefer-destructuring
     this.comment = revision.comments.split('\n')[0];
     this.tags =
       this.comment.search('Backed out') >= 0 ||
       this.comment.search('Back out') >= 0
         ? 'backout'
         : '';
+
+    if (this.comment.search('Backed out') >= 0) {
+      const backOutBugID = this.comment.match(/\(([^)]+)\)/)[1];
+      this.comment = `${backOutBugID} - ${this.comment.split('\n')[0]}`;
+    }
   }
 
+  toggleBugTooltip = async () => {
+    const bugComment = this.comment.split(' ');
+    const bugId = bugComment[1];
+    const { data, failureStatus } = await getData(
+      bugzillaBugsApi('bug', { id: bugId }),
+    );
+    return { data, failureStatus };
+  };
+
+  toggleTooltip = () => {
+    const { tooltipOpen } = this.state;
+    this.toggleBugTooltip();
```

Comment:
The return value of this function is where you should be getting the bug `summary` for the bug in question.  You then need to set that in the `state` so that the `tooltip` can display it.